lib/spoom/cli/commands/coverage.rb

```diff
@@ -25,6 +25,7 @@ def snapshot
         option :from, type: :string
         option :to, type: :string, default: Time.now.strftime("%F")
         option :save_dir, type: :string
+        option :bundle_install, type: :boolean, desc: "Execute `bundle install` before collecting metrics"
```

Comment:
It may be beneficial here to give an explanation about why a user would want to specify this: Talking about how this "fixes potential errors due to difference in Sorbet versions".

Reply:
Good point, I added a not in the `README`.